pycpfcnpj/calculation.py

```diff
@@ -1,47 +1,62 @@
 DIVISOR = 11
 
+CPF_WEIGHTS = ((10, 9, 8, 7, 6, 5, 4, 3, 2),
+              (11, 10, 9, 8, 7, 6, 5, 4, 3, 2))
+CNPJ_WEIGHTS = ((5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2),
+               (6, 5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2))
 
-def first_check_digit(number, weighs):
-    """ This function calculates the first check digit to
-        verify a cpf or cnpj vality.
 
-        :param number: cpf or cnpj number to check the first
-                       digit.  Only numbers.
+
+def calculate_first_digit(number):
+    """ This function calculates the first check digit of a
+        cpf or cnpj.
+
+        :param number: cpf (length 9) or cnpf (length 12) 
+            string to check the first digit. Only numbers.
         :type number: string
-        :param weighs: weighs related to first check digit
-                       calculation
-        :type weighs: list of integers
-        :returns: string -- the first digit
+        :returns: string -- the first digit, X if invalid size
 
     """
 
     sum = 0
-    for i in range(len(weighs)):
-        sum = sum + int(number[i]) * weighs[i]
+    if len(number) == 9:
+        weights = CPF_WEIGHTS[0]
+    elif len(number) == 12:
+        weights = CNPJ_WEIGHTS[0]
+    else:
+        return 'X'
+
+    for i in range(len(number)):
+        sum = sum + int(number[i]) * weights[i]
     rest_division = sum % DIVISOR
     if rest_division < 2:
         return '0'
     return str(11 - rest_division)
 
 
-def second_check_digit(updated_number, weighs):
-    """ This function calculates the second check digit to
-        verify a cpf or cnpj vality.
+def calculate_second_digit(number):
```

Comment:
Same comment in `calculate_first_digit `

Reply:
Fixed in https://github.com/matheuscas/pycpfcnpj/pull/19/commits/4a2080a4989c99070e21298527003fd4958da81d